tests/test_timeline_ui.py

```
@@ -0,0 +1,298 @@
+"""Tests for timeline_ui module."""
+
+import io
```

Flake8 (7.2.0)

[error] 'io' imported but unused

(F401)